gradle-plugins/src/main/kotlin/io.opentelemetry.instrumentation.muzzle-check.gradle.kts

```diff
@@ -167,18 +147,21 @@ if (hasRelevantTask) {
   afterEvaluate {
     var runAfter = muzzle
 
+    // the project repositories need to be retrieved after evaluation, before that the list is just empty
+    val projectRepositories = getProjectRepositories(project)
```

Comment:
Turns out that the project repositories weren't taken into account (they need to be retrieved `afterEvaluate`, before that the list is just empty); and since restlet is not available in maven central, muzzle check did nothing.